.github/workflows/go.yml

```diff
@@ -16,30 +30,20 @@ jobs:
     - name: Set up Go 1.x
       uses: actions/setup-go@v2
       with:
-        go-version: ^1.13
+        go-version: ^1.14
```

Comment:
Could change this to `^1.16`.

Reply:
```suggestion
        go-version: ^1.16
```

Reply:
@ryancurrah Should I bump the version in go.mod to 1.16 as well?

Reply:
Ah sorry I didn't see this. It doesn't matter to much.